db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1164,6 +1164,7 @@ void DBImpl::NotifyOnCompactionBegin(ColumnFamilyData* cfd, Compaction* c,
   TEST_SYNC_POINT("DBImpl::NotifyOnCompactionBegin::UnlockMutex");
   {
     CompactionJobInfo info{};
+    info.cf_id = cfd->GetID();
     info.cf_name = cfd->GetName();
     info.status = st;
     info.thread_id = env_->GetThreadID();
```

Comment:
There seems to be an issue with the existing code, not this PR. That is, the `CompactionJobInfo` populating logic looks identical to the code in `BuildCompactionJobInfo()`. Do you mind checking if we can reuse `BuildCompactionJobInfo()`? If it turns out difficult, let us know and we can still take this as is.

Reply:
I have viewed the two routines side by side.  They appear to be identical except for the missing cod->GetID().  So yes, BuildCompactionJobInfo() could be used here.  Are you asking that I make the change in this PR? @ajkr

Reply:
Yes, I think it'd be good to make the change in this PR.